lib/modules/hosts/youtube.js

```diff
@@ -19,7 +21,7 @@ export default new Host('youtube', {
 			(/watch|embed|v/i).exec(split[0]) && split[1]; // watch/embed/v
 	},
 	handleLink(href, hash) {
-		let src = `https://www.youtube.com/embed/${hash}?enablejsapi=1&enablecastapi=1`;
+		let src = insertParams(`https://www.youtube.com/embed/${hash}`, { enablejsapi: 1, enablecastapi: 1 });
```

Comment:
I think you can remove these parameters since there's no messaging occurring (so `enablejsapi` is unnecessary) and `enablecastapi` is not documented anywhere